src/modules/createImg.js

```diff
@@ -0,0 +1,5 @@
+export default function createPokeImage(pokeID, containerDiv) {
+  const pokeImage = document.createElement('img');
+  pokeImage.src = `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/${pokeID}.png`;
+  containerDiv.append(pokeImage);
+}
```

Comment:
- [ ] Great job, :clap: but I think it will be better if you can use ES6 arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html). Kindly make this for all your code